marketplace_confg/chart/dicom-adapter/templates/application.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
   labels:
     app.kubernetes.io/name: "{{ .Release.Name }}"
   annotations:
+    kubernetes-engine.cloud.google.com/icon: >-
+      data:image/png;base64,{{ .Files.Get "logo.png" | b64enc }}
```

Comment:
does logo.png need to be included in this pr?

Reply:
Yeah it's useless here, removed